app/blueprints/application/forms.py

```diff
@@ -1,15 +1,50 @@
 from flask_wtf import FlaskForm
-from govuk_frontend_wtf.wtforms_widgets import GovSelect, GovTextInput
+from govuk_frontend_wtf.wtforms_widgets import GovSelect, GovSubmitInput, GovTextInput
 from wtforms import HiddenField, SelectField, StringField
+from wtforms.fields.simple import SubmitField
 from wtforms.validators import DataRequired
 
+from app.db.queries.application import get_all_template_forms
+
+SELECT_A_TEMPLATE = "Select a template"
```

Comment:
Please just inline this text. It doesn't feel very helpful to have just one random string defined as a variable when all the rest are inline.

Reply:
it was added to fix sonar maintainability issue. will revert it back